data/conversations.txt

```diff
@@ -39,6 +44,11 @@ conversation "flight check: no steering energy"
 	`You've heard enough horror stories about the sort of accidents that happen when starship systems malfunction to be completely convinced of the necessity of a pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check.`
 	`	It is all going smoothly until it comes time to fire up the steering engines for a test run... and you discover that your generator does not produce enough power to drive them. You'll have to either install some batteries, or switch to a more powerful generator.`
 
+conversation "flight check: solar power warning"
+	scene "scene/engine2"
+	`You've heard enough horror stories about the sort of accidents that happen when starship systems malfunction to be completely convinced of the necessity of a pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check. Engines: check.`
+	`	Everything seems fine until you notice an amber warning light: your ship's computers have detected that your solar collectors may not be able to power your thrusters or steering too far from a star. You can launch your ship, but you run the risk of becoming derelict if you fly too far from a system centre.`	
```

Comment:
Changed it. Find it really difficult to type American English, but you're absolutely right.